### PR TITLE
[dash] Moved lotsa files around as per the IA

### PR DESCRIPTION
**Pull request by Sfshaza**
Also, set up a redirect for the box constraints link.  (An error message from one of the tools links to this file.)

I verified a nice clean link check.

@chalin

### CODE REVIEW

**Comment by Sfshaza** [comment]
@chalin, it's not clear why there was a Travis error. It just says "write error".

File: src/_includes/head.html [hunk 1]
```diff
@@ -28,12 +28,6 @@
   <meta property="og:image" content="{{og_image_path | absolute_url}}">
 
   <link href="https://fonts.googleapis.com/css?family=Google+Sans:400|Roboto:300,400,500|Source+Code+Pro|Material+Icons" rel="stylesheet">
-  {% asset main.css %}
-  {% for css in page.css -%}
-    {% asset '{{css}}' %}
-  {% endfor %}
-  {% asset main.js %}
-  <link rel="canonical" href="{{ page_url }}">
```
Comment (chalin) [inline src/_includes/head.html]: Were these the lines that you inadvertently removed? They should be restored.

File: src/_includes/setup/get-sdk-win.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 1. Download the following installation bundle to get the latest beta release of the
 Flutter SDK (for other release channels, and older builds, see the [SDK
-archive](/sdk-archive/) page.):
+archive](/get-started/sdk-archive) page.):
```
Comment (chalin) [inline src/_includes/setup/get-sdk-win.md]: Why place the sdk-archive under get-started? /cc @filiph 

On www we have this under the equivalent of what would here be `/development/tools/sdk/archive`. Use that instead?

File: src/_includes/head.html (hunk 1 above)
Comment (Sfshaza) [reply src/_includes/head.html]: Yes, that was some of the lines, not all.

File: src/_includes/setup/get-sdk-win.md (hunk 2 above)
Comment (Sfshaza) [reply src/_includes/setup/get-sdk-win.md]: sgtm

**Comment by Sfshaza** [comment]
There's a broken link in the codelab, Patrice. I'm going to go ahead and merge this, since Fernando needs these changes, but that broken link needs to be fixed.

File: src/_includes/head.html (hunk 1 above)
Comment (chalin) [reply src/_includes/head.html]: Thanks for recovering part of the lines but this is still missing `{% asset main.js %}`. I'll add it back in a followup PR.

**Comment by chalin** [comment]
> There's a broken link in the codelab, Patrice. I'm going to go ahead and merge this, since Fernando needs these changes

Thanks for merging so that we can work on the files.

> but that broken link needs to be fixed.

Done, #1305. Travis is green now.

File: src/_includes/head.html (hunk 1 above)
Comment (chalin) [reply src/_includes/head.html]: Done #1306

**Comment by Sfshaza** [comment]
Sorry, Patrice, and thanks!

**Pull request merged**